pkg/controllers/provisioningv2/rke2/machinestatus/machinestatus.go

```diff
@@ -32,7 +32,8 @@ import (
 )
 
 const (
-	Provisioned = condition.Cond("Provisioned")
+	Provisioned         = condition.Cond("Provisioned")
+	InfrastructureReady = condition.Cond("InfrastructureReady")
```

Comment:
Per offline, attempt to use the upstream CAPI consts here

Reply:
There is a Condition constant for `InfrastructureReady`, but not for `Provisioned`.